VERSION

```diff
@@ -1 +1 @@
-1.3.0
+1.3.1
```

Comment:
are we going to publish this gem? if it will be published privately, I'd suggest having more custom version (e.g. `1.3.0-vinted-p1`) to avoid unintended updates from public repo

Reply:
I thought of simply declaring `git: 'https://github.com/vinted/gssapi'`, as we have with `ruboozie`